src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -609,7 +609,7 @@
 helps['resource link create'] = """
     type: command
     short-summary: Create a new link between resources.
-    long-summary: A link-id is of the form /subscriptions/{SubID}/resourceGroups/{ResourceGroupID}/{ProviderNamespace}/{ResourceType}/{ResourceName}/Microsoft.Resources/links/{LinkName}
+    long-summary: A link-id is of the form /subscriptions/{SubID}/resourceGroups/{ResourceGroupID}/providers/{ProviderNamespace}/{ResourceType}/{ResourceName}/providers/Microsoft.Resources/links/{LinkName}
```

Comment:
/providers is missing for other resource link commands below as well: resource link show, resource link update, resource link delete. Can you please fix them as well? Thanks.